### ISSUE TITLE
ord wallet --name create is broken

### ISSUE THREAD

**Issue by dannybabbev** [ordinals/ord#2942]
After the update that introduced the `wallet --name {walletName}` instead of `--wallet {walletName} wallet` command, the wallet create functionality is broken.

`ord wallet --name block999 create` fails with:

```
[2024-01-03T16:07:05Z INFO  ord::options] Connecting to Bitcoin Core at bitcoin:8332/
error: JSON-RPC error: RPC error response: RpcError { code: -19, message: "Wallet file not specified (must request wallet RPC through /wallet/<filename> uri-path).", data: None }
```

**Comment by so7ow** [comment]
What version of ord are you referring to?

**Comment by dannybabbev** [comment]
ord 0.13.1

**Comment by so7ow** [comment]
```
ord13 % ./ord --version
ord 0.13.1
ord13 % ./ord --help
Usage: ord [OPTIONS] <COMMAND>

Commands:
  balances  List all rune balances
  decode    Decode a transaction
  epochs    List the first satoshis of each reward epoch
  find      Find a satoshi's current location
  index     Index commands
  list      List the satoshis in an output
  parse     Parse a satoshi from ordinal notation
  preview   Run an explorer server populated with inscriptions
  runes     List all runes
  server    Run the explorer server
  subsidy   Display information about a block's subsidy
  supply    Display Bitcoin supply information
  teleburn  Generate teleburn addresses
  traits    Display satoshi traits
  wallet    Wallet commands
  help      Print this message or the help of the given subcommand(s)

Options:
      --bitcoin-data-dir <BITCOIN_DATA_DIR>
          Load Bitcoin Core data dir from <BITCOIN_DATA_DIR>.
      --bitcoin-rpc-pass <BITCOIN_RPC_PASS>
          Authenticate to Bitcoin Core RPC with <RPC_PASS>.
      --bitcoin-rpc-user <BITCOIN_RPC_USER>
          Authenticate to Bitcoin Core RPC as <RPC_USER>.
      --chain <CHAIN_ARGUMENT>
          Use <CHAIN>. [default: mainnet] [possible values: mainnet, testnet, signet, regtest]
      --config <CONFIG>
          Load configuration from <CONFIG>.
      --config-dir <CONFIG_DIR>
          Load configuration from <CONFIG_DIR>.
      --cookie-file <COOKIE_FILE>
          Load Bitcoin Core RPC cookie file from <COOKIE_FILE>.
      --data-dir <DATA_DIR>
          Store index in <DATA_DIR>.
      --db-cache-size <DB_CACHE_SIZE>
          Set index cache to <DB_CACHE_SIZE> bytes. By default takes 1/4 of available RAM.
      --first-inscription-height <FIRST_INSCRIPTION_HEIGHT>
          Don't look for inscriptions below <FIRST_INSCRIPTION_HEIGHT>.
      --height-limit <HEIGHT_LIMIT>
          Limit index to <HEIGHT_LIMIT> blocks.
      --index <INDEX>
          Use index at <INDEX>.
      --index-runes
          Track location of runes. RUNES ARE IN AN UNFINISHED PRE-ALPHA STATE AND SUBJECT TO CHANGE AT ANY TIME.
      --index-sats
          Track location of all satoshis.
  -r, --regtest
          Use regtest. Equivalent to `--chain regtest`.
      --rpc-url <RPC_URL>
          Connect to Bitcoin Core RPC at <RPC_URL>.
  -s, --signet
          Use signet. Equivalent to `--chain signet`.
  -t, --testnet
          Use testnet. Equivalent to `--chain testnet`.
      --wallet <WALLET>
          Use wallet named <WALLET>. [default: ord]
  -h, --help
          Print help
  -V, --version
          Print version

```          
help still says to use --wallet in 13.1... am i missing a change?

**Comment by ordinariusprof** [comment]
@so7ow change is introduced in https://github.com/ordinals/ord/commit/d791800fa438469ac8102f8f972e2a1e99b16bd4 where `--wallet` command is replaced with `wallet --name`. technically it went into v0.14. but issue is there.

**Comment by so7ow** [comment]
> @so7ow change is introduced in [d791800](https://github.com/ordinals/ord/commit/d791800fa438469ac8102f8f972e2a1e99b16bd4) where `--wallet` command is replaced with `wallet --name`. technically it went into v0.14. but issue is there.

got it. thx.

**Comment by raphjaph** [comment]
Thanks! We passed in the wrong client to the descriptor import function. Tests didn't catch it because they were running against a mocked Bitcoin Core. Should be fixed in #2943

**Comment by raphjaph** [comment]
Made a new release as well
